docs/gaps/gaps.tex

```diff
@@ -0,0 +1,286 @@
+agents can query the previous (or, eventually, expected) demand of fuel and adjust their throughputs to meet the demand. 
+The market-peeking capability is essential for demand-prediction models.
+
+Moreover, with the market-peeking capability, support facilities can estimate adjust their
+capacities by estimating
+the demand of fuel/fissile material proportional to power demand (`power' is a market commodity in \Cyclus).
+
+\subsection{A Need For Inter-Agent Feedback}
+Inter-agent feedback means that agents can communicate with each other and change its parameters accordingly. 
+For example, an increase in fabrication capacity, should cause an increase in reprocessing capacity. Also,
+if a reactor is shutdown due to lack of fuel, this should cause a flag to increase fabrication capacity,
+or `investigate'
+This would be problematic in parallel to the market-peeking capability, since there would be more than one algorithm
+that changes the throughput of an agent. 
+
+\subsection{Irresponsible deployment}
+The \textsc{DeployInst}\xspace model deploys prototypes in a user-defined time period, even though there is not 
+enough fuel to operate the reactor. The deployed reactor is remains idle if the fuel supply is short. Deploying new reactors
+without sufficient fissile material to support it can cause other already-deployed reactors without fuel, causing
+a cascade of shutdowns. Using the market-peeking or feedback capability, new reactors should be deployed only
+if there is sufficient fuel to support the startup of the reactor as well as currently deployed reactors. 
+
+
+\section{Additional Possible Improvements}
+Though not critical to the current goal, having the following capabilities can increase
+the accuracy of simulations.
+
+
+\subsection{Blanket capability for \textsc{Reactor}\xspace}
+For \glspl{SFR}, the blanket and the driver have different effective fuel residence times, meaning that
+they should (ideally) be discharged and loaded separately. However, with current capabilities, the option is the average
+the mass and composition of driver and blanket to treat is as one commodity.
+
+\subsection{Depletion Calculations}
+Depletion calculations are done outside of the simulation. However, if advanced reactor models like \textsc{Cyborg}\xspace
+or \textsc{Bright-Lite}\xspace are utilized, there could be more accurate depletion calculations and 
+perhaps dynamic Breeding Ratio modeling.
+
+\subsection{\textsc{Reactor}\xspace Decommissioning Behavior}
+The \Cycamore \textsc{Reactor}\xspace, when decommissioned mid-cycle, transmutes half of
+its current assemblies, and discharges all its core as waste commodity. This is in order to model the average
+composition of discharged assemblies in different stages. An alternative would
+be to have recipes for each stage (ie. for four batch, there would be three spent fuel recipes).
+
+
+\bibliographystyle{abbrv}
+\bibliography{./bibliography.bib}
```

Comment:
The filepath "./" is redundant, as is the file extension ".bib"


Also, these references are missing.
```
Latexmk: List of undefined refs and citations:
  Citation `djokic_application_2015' on page 1 undefined on input line 80
  Citation `wigeland_nuclear_2014' on page 1 undefined on input line 55
  Citation `wigeland_nuclear_2014' on page 1 undefined on input line 84
  Reference `diag:hd' on page 1 undefined on input line 194
  Reference `fig:mox' on page 1 undefined on input line 155
  Reference `fig:sfr' on page 1 undefined on input line 155
  Reference `tab:es' on page 1 undefined on input line 56
```